.babelrc

```diff
@@ -1,10 +1,5 @@
 {
   "presets": [
-    ["latest", {
```

Comment:
[Personal preference] I set the babel-loader config options with .babelrc. Does this .babelrc conflict with the test suite? If so maybe we can just add another one inside the src/app folder so that babel-loader can pick up those transform options there.